env0/resource_environment_output_configuration_variable.go

```diff
@@ -0,0 +1,319 @@
+package env0
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type EnvironmentOutputConfigurationVariableParams struct {
+	Name                      string
+	OutputEnvironmentId       string
+	OutputSubEnvironmentAlias string
+	OutputName                string
+	Scope                     string
+	ScopeId                   string
+	Description               string
+	Type                      string
+	IsReadOnly                bool
+	IsRequired                bool
+
+	outputEnvironmentProjectId string
+}
+
+type EnvironmentOutputConfigurationVariableValue struct {
+	OutputName          string `json:"outputName"`
+	ProjectId           string `json:"projectId"`
```

Comment:
projectId can be removed from here :) we refactored that part and removed it from this schema